.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Black code reformatting
```

Comment:
There's a decent chance that we'll have another reformat due to Black and it would be nice if this comment directly included the context from the commit message.
```suggestion
# Nov. 2020 commit that mass-reformatted using Black
```